ingestion-beam/src/main/java/com/mozilla/telemetry/republisher/RepublisherOptions.java

```diff
@@ -93,19 +93,13 @@
 
   void setPerDocTypeDestination(String value);
 
-  @Description("A comma-separated list of namespaces that should be republished to individual"
-      + " topics")
-  List<String> getPerNamespaceEnabledList();
+  @Description("A JSON-formatted map of document namespaces to output topic names (assuming"
+      + " --outputType=pubsub) for per-namespace sampling; the verbose map representation is used"
+      + " here to support cases where the destination topics are heterogeneous and may live in"
+      + " different projects")
+  Map<String, String> getPerNamespaceDestinations();
```

Comment:
I don't think it's worth blocking this PR, but to reduce the amount of repetition here maybe we could accept `{"<destination_pattern>":["<namespace>"]}` e.g.
```json
{
  "project/telemetry/topic/per-namespace-${document_namespace}":[
    "telemetry",
    "glean"
  ],
  "project/pocket/topic/namespace-pocket":[
     "pocket"
  ]
}
```

Reply:
Hmmm... Interesting. I will think on this.

Reply:
Broke this out to https://github.com/mozilla/gcp-ingestion/issues/677